src/vos/vos_io.c

```diff
@@ -214,6 +222,10 @@ vos_ioc_create(daos_handle_t coh, daos_unit_oid_t oid, bool read_only,
 		goto error;
 	}
 
+	D_ALLOC_ARRAY(ioc->ic_biov_dcbs, 16);
+	ioc->ic_biov_dcb_at = 0;
+	ioc->ic_biov_dcb_nr = 16;
```

Comment:
please define a macro for it if you use the same value for more than once

Reply:
rearranged code so the value is only used once.